src/analysis/modals/ComputeModal/GcpComputeModal/GcpComputeModal.test.js

```diff
@@ -699,21 +702,13 @@ describe('GcpComputeModal', () => {
 
     // Act
     await act(async () => {
-      await render(h(GcpComputeModalBase, defaultModalProps));
-
-      const selectMenu = await screen.getByLabelText('Application configuration');
-      await userEvent.click(selectMenu);
+      render(h(GcpComputeModalBase, defaultModalProps));
+      const selectMenu = screen.getByLabelText('Application configuration');
+      await user.click(selectMenu);
       const selectOption = await screen.findByText(hailImage.label);
-      await userEvent.click(selectOption);
-
-      const computeTypeSelect = await screen.getByLabelText('Compute type');
-      await userEvent.click(computeTypeSelect);
-
-      const sparkSingleNodeOption = await screen.findByText('Spark single node')[0];
-      await userEvent.click(sparkSingleNodeOption);
```

Comment:
Were this line steps important for this test?

Reply:
No, the modal was actually simplified and changing one drop down option results another drop down option being re-selected for you. In other words, these steps were invalid after this change